attribution-graphs-frontend/attribution_graph/cg.css

```
@@ -0,0 +1,772 @@
+
+        svg{
+            z-index: -999999999;
+        }
+
+        .weight-label{
+            /* text-shadow: #fff .2px 0 .2px, #fff -.2px 0 .2px, #fff 0 .2px .2px, #fff 0 -.2px .2px; */
+            font-size: 9px;
+            font-family: system-ui;
+            text-anchor: middle;
+            alignment-baseline: bottom;
+            dy: -.5em;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)